### PR TITLE
Don't export module

### PR DESCRIPTION
**Pull request by shosti**
It's a major pain maintaining both, and it complicates the build on the
schrute side. As a hacky alternative, export the code directly in
bootstrap.

@rainforestapp/tester-product @ukd1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,6 @@ Clone this repo
 1. `npm install`
 2. `npm test`
 
-### 4. Before committing
-
-`npm run build` (and commit the changes to lib/)
```
Comment (pscohn) [inline README.md]: I'm a little confused, when/where does the src get built now?

Comment (sweeneyapps) [inline README.md]: https://github.com/rainforestapp/tester-chrome-extension/blob/nuke-module/deploy

Comment (shosti) [inline README.md]: @pscohn it was previously getting build to build/main.js (for the browser) *and* lib/*.js (as a node module). The module was really just for the fake plugin on schrute. This PR just drops the lib/ stuff; schrute fake plugin will just import the browser version (https://github.com/rainforestapp/schrute/pull/54).

**Comment by pscohn** [comment]
🚢

**Pull request merged**